spec/support/touch.rb

```diff
@@ -0,0 +1,3 @@
+module Touch
+  def touch(*toucheables); end
+end
```

Comment:
The antonym for https://en.wikipedia.org/wiki/The_Intouchables :smiley:

Reply:
Amazing movie btw : )